src/store/modules/notices.js

```diff
@@ -33,6 +33,7 @@ const actions = {
     }
   },
   createNotice: async ({ commit }, params = {}) => {
+    commit(SET_CREATE_ERRORS, []);
```

Comment:
what is SET_CREATE ERRORS?

Reply:
Action type (redux-same flow), if interested read about store in js apps (redux, vuex)